mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperTest.java

```diff
@@ -141,6 +141,26 @@ void setUp() {
                 .asUserBound();
         }
 
+        @Test
+        void createShouldBeConsistentWhenFailToPersistMailbox() {
```

Comment:
It works?

BTW can I create after a failed create?

Reply:
> It works?

Yep

> BTW can I create after a failed create?

Let me add a test for that

Reply:
I suspect it works because `create` do not rely on `inboxId`, you might need to rework the test.

Reply:
regarding the first test, `findMailboxByPath` gets the mailboxId first from `mailboxPath` then tries to fetch the mailbox with the id -> `MailboxNotFoundException` is normal 

To check this, need the `create after failed create` test (that fails)

Note though: the test was still wrong as doQuietly is sneaky... need to set the mailboxId to null first before the create.